docs/spec/distribution/state.md

```diff
@@ -0,0 +1,108 @@
+## State
+
+### Global
+
+All globally tracked parameters for distribution are stored within
+`Global`.  Rewards are collected are added to the reward pool and
+distributed to validators/delegators from here. 
+
+Note that the reward pool is held decimal coins (`DecCoins`) to allow 
+for fractions of coins to be received from operations like inflation. 
+When coins are distributed from the pool they are truncated back to 
+`DecCoins` which is non-decimal. 
+
+ - Global:  `0x00 -> amino(global)`
+
+```golang
+// coins with decimal 
+type DecCoins []DecCoin
+
+type DecCoin struct {
+    Amount sdk.Dec
+    Denom  string
+}
+
+type Global struct {
+	PrevBondedTokens  sdk.Dec  // bonded token amount for the global pool on the previous block 
+    Adjustment        sdk.Dec  // global adjustment factor for lazy calculations
+    Pool              DecCoins // funds pool for all validators
+    PrevReceivedPool  DecCoins // funds added to the pool on the previous block
+    EverReceivedPool  DecCoins // total funds ever added to the pool 
+    CommunityFund     DecCoins // pool for community funds
+}
+```
+
+### Validator Distribution
+
+Validator Distribution information for the relevant validator is updated: each
+time delegations to a validator are updated, a validator successfully proposes
+a block and receives (receiving a reward), any delegator withdraws from a
+validator, the validator withdraws. 
+
+ - ValidatorDistribution:  `0x02 | ValOwnerAddr -> amino(validatorDistribution)`
+
+```golang
+type ValidatorDistribution struct {
+    CommissionWithdrawalHeight   int64    // last time this delegation withdrew rewards
```

Comment:
This delegation or this validator?

Reply:
changed to `last time this validator withdrew commission`